controllers/idler/idler_controller.go

```diff
@@ -110,7 +115,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 	// Find the earlier pod to kill and requeue. Otherwise, use the idler timeoutSeconds to requeue.
 	nextTime := nextPodToBeKilledAfter(logger, idler)
 	if nextTime == nil {
-		after := time.Duration(idler.Spec.TimeoutSeconds) * time.Second
+		after := time.Duration(RequeueTimeThreshold) * time.Second
```

Comment:
But it would use the 300 second timeout only if nextTime == nil, right? So if it's set to 7 hours for example then we will wait 7h. I think we should use nextTime only if it's shorter than 300 seconds.

Reply:
right. Okay, let me make that change.

Reply:
done in [a9b2750](https://github.com/codeready-toolchain/member-operator/pull/630/commits/a9b27504aaa54ce3fd7aa66dad69abca3e0bf8f9)
To clarify - this would affect all workloads not just crashlooping ones

Reply:
sorry, fixed the condition now in d40b75b

Reply:
Yes, it will affect all the pods. So we are basically now requeueing every 5 minutes (or faster) for all pods. Which I think we have to do. Otherwise if the pod starts crash-loping, let's say in 10 minutes after the start we won't catch it because we will reconcile only in 12h.

Reply:
agreed, I know we discussed this, i just wanted to clarify in the PR for others reviewing as well.